### DISCUSSION TITLE
Manage multiple microlc services inside the Microfrontend Composer

### DISCUSSION THREAD

**Discussion by Noste92** [mia-platform/community#581] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

Microfrontend Composer

### Description

Hi,

I'd like to try this feature of the Microfrontend Composer, but I didn't find any way to add a second instance of Microlc to the Composer.

![image](https://github.com/user-attachments/assets/3651f9b1-b610-4f39-87e2-7bb6cd6a24b7)

Could you help me to find the right section of the documentation where I can find how to do that?

Thank you

**Comment by epessina** [comment, marked as answer]
Hi @Noste92 , as of today you can create multiple instances of [Microfronted Compose Toolkit application](https://docs.mia-platform.eu/docs/runtime_suite_applications/microfrontend-composer-toolkit/overview) from Mia-Platform marketplace. Each application will bring a new [micro-lc](https://github.com/micro-lc/micro-lc) instance (or, more precisely, a new [middleware](https://github.com/micro-lc/middleware) instance), that will be configurable from the Configurator.

To ease the process we will publish middleware on the marketplace as a standalone plugin, and we make the procedure clearer in the documentation.

**Reply by Noste92** [reply]
Hi @epessina, thanks for your answer. That clears things up.
